include/ServerAccept.hpp

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Jack Khuu, Justin Johnson, Lukas Joswiak
+
+#ifndef INCLUDE_SERVERACCEPT_HPP_
+#define INCLUDE_SERVERACCEPT_HPP_
+
+#include <string>
+#include <sstream>
+
+#include <boost/archive/text_oarchive.hpp>
+#include <boost/archive/text_iarchive.hpp>
+
+class ServerAccept {
+ public:
+  ServerAccept();
+  explicit ServerAccept(std::string local_port);
+
+  std::string GetLocalPort() const {
+    return local_port_;
+  }
+
+  std::string Encode() const;
+
+  static ServerAccept Decode(const std::string data);
+
+ private:
+  friend class boost::serialization::access;
+
+  template<class Archive> void serialize(Archive &ar,
+      unsigned int version) {
```

Comment:
is version needed here? if not, can you name it `unsigned int _ /* version */`. This is how they handle unused params at fb.